templates/simulations/assembly_addition.jinja2

```diff
@@ -1,20 +1,46 @@
-{% extends "simulations/base.jinja2" %}
+{% extends "simulations/demo.jinja2" %}
 
 {% block title %}IBM 704 Assembly Addition{% endblock %}
 
 {% block header %}IBM 704 Assembly Addition{% endblock %}
 
-{% block buttons %}
-    <button id="step_button">Step</button>
-    <button id="run_button">Run</button>
-    <button id="reset_button">Reset</button>
-{% endblock %}
-
 {% block description %}
-    Here's what's happening here:<br />
-    <b>CLA 4: </b> Clears the accumulator and stores the value at address 4 (which you'll see if you mouseover is 12) into the accumulator.<br />
-    <b>ADD 5: </b> Adds the value at address 5 (30) to the accumulator.<br />
-    <b>STO 6: </b> Stores the value of the accumulator into the register at address 6.<br />
-    <b>HTR: </b> Tells the computer to stop.<br />
-    If you run this, you'll see that register 6 has a fixed point value of 42.  The answer to life, the universe, and everything.  Everything as it should be.<br />
+    Welcome to the IBM 704 simulator!  Here we have a simple example to introduce you how it works.<br/>
```

Comment:
For the second sentence, I would remove `simple` because we don't know whether our readers will think this is simple or not (when writing code documentation, it is best to avoid judgement). It is also a little awkward in phrasing. I recommend this as a new second sentence: 

```
To learn how it works, we'll walk through an example. 
```